docs/pages/setup/admin/github-sso.mdx

```diff
@@ -34,10 +34,15 @@ Instructions for creating a GitHub OAuth app are available here:
 
 [Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
 
-## Step 2/3. Create a GitHub authentication connector
+## Step 2/5. Define a GitHub authentication connector
 
-Define a GitHub authentication connector by creating a file called `github.yaml`
-with the following content:
+Define a GitHub authentication connector by running the following command:
+
+```code
```

Comment:
In case a reader blindly copies and runs this command, I think we should include variables using the `${VARNAME?}` syntax. We can also include environment variable assignments for the client ID and client secret, plus a quick instruction to grab these from your newly created OAuth app

Reply:
Do we have an existing example of this approach in our docs?

Reply:
Here's one example: https://github.com/gravitational/teleport/blame/348b290cc7db363ebb76f3553a4625b11fe60a1e/docs/pages/kubernetes-access/getting-started.mdx#L77-L89